rotkehlchen/tests/api/test_user_evm_tokens.py

```diff
@@ -577,3 +577,30 @@ def test_user_tokens_delete_guard(rotkehlchen_api_server):
         contained_in_msg=expected_msg,
         status_code=HTTPStatus.CONFLICT,
     )
+
+
+def test_non_ethereum_token(rotkehlchen_api_server):
+    response = requests.put(
+        api_url_for(
+            rotkehlchen_api_server,
+            'ethereumassetsresource',
+        ),
+        json={
+            'token': {
+                'name': 'Some random name',
+                'symbol': 'XYZ',
+                'cryptocompare': None,
+                'coingecko': None,
+                'started': 1599646888,
+                'swapped_for': None,
+                'address': '0xC88eA7a5df3A7BA59C72393C5b2dc2CE260ff04D',
+                'chain': 'binance',  # important that is not `ethereum` here
+                'token_kind': 'erc20',
+                'decimals': 18,
+                'protocol': 'my-own-protocol',
+                'underlying_tokens': None,
+            },
+        },
+    )
+    result = assert_proper_response_with_result(response)
+    assert EvmToken(result['identifier']).protocol == 'my-own-protocol'
```

Comment:
since it is an addition I would also check that the `chain`, `decimals`, `name`, `symbol` and `address` at least are the ones expected

Reply:
Does it make sense? They are stored in the same table as protocol, and this is checked in other tests. 

This one just makes sure that the addition happened and the token is in the db.

Reply:
imo it does since is to make sure that the different value for chain is stored correctly. I believe it could be also good to check what happens if you insert an already existing address but in a different chain (some tokens have the same address in multiple chains)

Reply:
but is not about conflict. Is just because ethereum was a special case before and with that check we protect us against the kind of bug you fixed where we assumed ethereum in the identifier. So checking the chain is to verify that again ethereum is not a special value and the name/symbol thing is to verify that there is no special check that skips the value or does a different thing

Reply:
Didn't really get the reasoning since it sounds like something hypothetical (we don't have any name are symbol checks that depend on the chain afaik), but it is not difficult, so will add these checks

Reply:
there are special checks for things like NFTs where the name/symbol is special based on the identifier. My knowledge is that with unit testing you should check against things that could break. In this case if you say we don't have a logic that modifies name/symbol you are assuming things about the code and your tests are no longer testing but assuming things about the code